front/components/assistant/AssistantDetails.tsx

```diff
@@ -385,7 +385,17 @@ export function AssistantDetails({
     agentConfigurationId: assistantId,
   });
 
+  const isGlobalAgent = useMemo(() => {
+    return Object.values(GLOBAL_AGENTS_SID).includes(
+      agentConfiguration?.sId as GLOBAL_AGENTS_SID
+    );
+  }, [agentConfiguration?.sId]);
```

Comment:
imo we should not use a useMemo here , computational cost is negligeable

```suggestion
const isGlobalAgent = Object.values(GLOBAL_AGENTS_SID).includes(
  agentConfiguration?.sId as GLOBAL_AGENTS_SID
);
```